src/Components/setup/SetupApp.js

```diff
@@ -10,7 +10,8 @@ const SetupConfig = () => {
     const requestInterceptor = axiosInstance.interceptors.request.use(
       (config) => {
         if (!config.params || !config.params.sessionID) {
-          console.log('Unauthorized. Please login again.');
+          const currentRoute = location.pathname;
+
           if (currentRoute !== '/') {
```

Comment:
create utility function to identify which routes need authentication and then handle redirect like so.